packages/core/src/api/blockManipulation/commands/splitBlock/splitBlock.ts

```diff
@@ -38,7 +44,7 @@ export const splitBlockCommand =
         newBlockContentPos,
         newBlockContentPos + 1,
         newBlockContent.content.size > 0
-          ? new Slice(Fragment.from(newBlockContent), depth + 2, depth + 2)
+          ? new Slice(Fragment.from(newBlockContent), depth + 3, depth + 3)
```

Comment:
I'm not sure about the depths we pass to the slice. I think we should just pass 0, 0.

See: 
https://prosemirror.net/docs/ref/#model.Slice.constructor

and

https://discuss.prosemirror.net/t/what-is-openstart-and-openend-use-for/3999